sdk/mixedreality/azure-mixedreality-authentication/azure/mixedreality/authentication/_shared/aio/mixed_reality_token_credential.py

```diff
@@ -0,0 +1,67 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import TYPE_CHECKING
+
+from .static_access_token_credential import StaticAccessTokenCredential
+from ...aio._client_async import MixedRealityStsClient
```

Comment:
in all the aoi shared code this should refer to azure.mixedreality.authentication.aio since the relative import will not work when we copy out the code

Reply:
Sure!